src/modules/systemlib/mixer/mixer.h

```diff
@@ -184,6 +184,19 @@ class __EXPORT Mixer
 	 */
 	virtual void			groups_required(uint32_t &groups) = 0;
 
+	/**
+	 * @brief      Update slew rate parameter. This tells the multicopter mixer
```

Comment:
I'm still confused about the inheritance model here. And the comment about this being a multirotor mixer seems wrong.